BigRedButton.js

```diff
@@ -55,7 +55,11 @@ function BigRedButton(index)
 util.inherits(BigRedButton, events.EventEmitter);
 
 BigRedButton.prototype.askForStatus = function() {
-   this.hid.write(cmdStatus);
+   try {
+      this.hid.write(cmdStatus);
+   } catch(e) {
+      this.close();
+   }
```

Comment:
This didn't show up in my cases before as it's a race condition to see if `write` or `read` gets called first. So this never happened but did later.